pkgs/development/libraries/ncurses/default.nix

```diff
@@ -48,20 +48,18 @@ stdenv.mkDerivation (finalAttrs: {
         "/usr/share/terminfo" # upstream default, probably all FHS-based distros
         "/run/current-system/sw/share/terminfo" # NixOS
       ]}"
-  ];
+  ] ++ lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) "--with-build-cc=${stdenv.cc.targetPrefix}cc";
```

Comment:
```suggestion

  depsBuildBuild = [ stdenv ];
```

Reply:
`stdenv` is spliced so it works here.  Unfortunately `stdenv.cc` isn't spliced :frowning:

Reply:
If you make this change I will close https://github.com/NixOS/nixpkgs/pull/265759/files and merge this instead (after the 23.11 branch-off for `staging`, which happens in about a week)

Reply:
https://github.com/NixOS/nixpkgs/pull/265162#discussion_r1382748821

Reply:
Nevermind.